experimental/sql/test/projection_pushdown_tests/extended_case.xdsl

```diff
@@ -0,0 +1,70 @@
+// RUN: rel_opt.py -p projection-pushdown %s | filecheck %s
+
+module() {
```

Comment:
This query has basically the structure of Q6, showing that we can reach the optimization level we want atm.

Notice that the "d" of the input table gets projected out properly, as we want it to.